app.js

```diff
@@ -1,25 +1,48 @@
-const express = require('express');
+const express = require('express'); // requerir express
 
-const hbs = require('hbs');
-const path = require('path');
+const hbs = require('hbs'); //requerir hbs
+const path = require('path');  // crea paths
 const PunkAPIWrapper = require('punkapi-javascript-wrapper');
 
-const app = express();
+const app = express(); //instanciar applicacion de express
 const punkAPI = new PunkAPIWrapper();
 
 app.set('view engine', 'hbs');
 app.set('views', path.join(__dirname, 'views'));
-
+punkAPI.getRandom().then(elm => console.log(elm))
```

Comment:
```suggestion
punkAPI.getRandom().then(elm => console.log(elm))
```
Intenta hacer estas comprobaciones solo donde las necesites, en este caso dentro de la ruta correspondiente!